documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// assembly-config-mirrormaker2.adoc
+
+[id='con-config-mirrormaker2-connectors-{context}']
+= Connector configuration
+
+[role="_abstract"]
+Use Mirrormaker 2.0 connector configuration for the internal connectors that orchestrate the synchronization of data between Kafka clusters.
+
+The following table describes connector properties and the connectors you configure to use them.
+
+.MirrorMaker 2.0 connector configuration properties
+[cols="4a,2,2,2",options="header"]
+|===
+
+|Property
+|sourceConnector
+|checkpointConnector
+|heartbeatConnector
+
+
+|admin.timeout.ms:: Timeout for admin tasks, such as detecting new topics. Default is `60000` (1 minute).
+|✓
+|✓
+|✓
+
+|consumer.poll.timeout.ms:: Timeout when polling the source cluster. Default is `1000` (1 second).
+|✓
+|✓
+|
+
+|offset-syncs.topic.location:: The location of the `offset-syncs` topic. Default is the _source_ cluster.
```

Comment:
Should we say this can be set to `source` (default) or `target`? Also setting it to `target` allows to only grant read access to the source cluster to MirrorMaker.

Reply:
Updated to:

> offset-syncs.topic.location:: The location of the `offset-syncs` topic, which can be the `source` (default) or `target` cluster. If you set the property to `target`, read-only access to the source cluster is granted to MirrorMaker.

Reply:
This is not quite right, setting this value does not grant any access! It's just that if users set this configuration to `target`, then they only need to grant read-only access to MirrorMaker on the source cluster. Otherwise, with the default value, MirrorMaker needs both read and write access on the source cluster. 

Maybe it's not the best place to explain this, do we have a section on MirrorMaker permissions?